packages/cherry-pick-bot/src/bot.ts

```diff
@@ -142,6 +145,9 @@ export = (app: Probot) => {
       repo,
       [pullRequest.sha!],
       targetBranch,
+      remoteConfig.preservePullRequestTitle
+        ? `cherry-pick ${pullRequest.title} (#${pullRequest.number})`
```

Comment:
This doesn't match conventional commit message.

How about to preserve the original title and add an indicator for being cherry-pick in the pull request body?

Reply:
Fixed